internal/zinc-core/src/main/scala/sbt/internal/inc/Incremental.scala

```diff
@@ -626,7 +626,7 @@ private final class AnalysisCallback(
   private[this] val objectApis = new TrieMap[String, ApiInfo]
   private[this] val classPublicNameHashes = new TrieMap[String, Array[NameHash]]
   private[this] val objectPublicNameHashes = new TrieMap[String, Array[NameHash]]
-  private[this] val usedNames = new RelationBuilder[String, UsedName]
+  private[this] val usedNames = new TrieMap[String, ConcurrentSet[UsedName]]
```

Comment:
This is a very late pong to this [ping](https://github.com/sbt/zinc/pull/764#issuecomment-817122949) from @dotta